CIP-0000.md

```diff
@@ -0,0 +1,198 @@
+- - - -
+lang: en
+cip: 0000
+title: Celo Improvement Proposal (CIP) Process
+status: Draft
+type: Meta
+discussions-to:
+author: Yaz Khoury (@YazzyYaz), James Prestwich (@prestwich), Pranay Mohan (@pranaymohan), Mariano Cortesi (@mcortesi)
+created: 2020-09-22
+license: CC0 1.0
+- - - -
+
+# Abstract
+A Celo Improvement Proposal (CIP) describe standards for the Celo platform, including the core protocol specifications, SDK, and contract standards. It is a design document providing information to the Celo community, or describing a new feature for Celo or its processes or environment, a rationale for the proposal, detailed solution including technical specifications, and, if any, a list of potential risks. 
+
+The CIP proposer, the **author** is responsible for building consensus within the community and documenting whether there is rough consensus on their proposal. 
+
+Because the CIPs are maintained as text files in a versioned repository, their revision history is the historical record of the feature proposal.
+
+Participation in the Celo project and the CIP process is informed by the Celo [Community Tenets](https://celo.org/community) and subject to the  [Code of Conduct](https://celo.org/code-of-conduct).
+
+# CIP Types
+There are three types of CIP:
+
+* A **Standards Track CIP** describes any change that affects most or all Celo implementations, such as a change to the network protocol, a change in block or transaction validity rules, proposed application standards/conventions, or any change or addition that affects the interoperability of applications using Celo. Furthermore Standard CIPs can be broken down into the following categories. Standards Track CIPs consist of two parts, a design document and implementation.
```

Comment:
There are cases of Meta tracks that aren't describing standards but a collection of standards, as in the case of referring to Hardfork Implementations from Ethereum (EIPs used) as references.

I'd like to keep Meta track because we will need to use those to refer to all adopted EIPs.

Informational ones aren't used as much, more as a nice-to-have but maybe useful in case they need to be used down the line. I am not strongly opinionated on Informational but Meta is very useful.

Reply:
OK thanks for the explanation - SGTM